velox/exec/tests/ArrowStreamTest.cpp

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec::test;
+
+class ArrowStreamTest : public OperatorTestBase {
+ protected:
+  /// A mock Arrow reader returns an Arrow array for each call of getNext.
+  class ArrowReader {
+   public:
+    ArrowReader(
+        std::shared_ptr<memory::MemoryPool> pool,
+        const std::vector<RowVectorPtr>& vectors,
+        const TypePtr& type,
+        bool markGettingNextFailed = false /* For test only.*/,
```

Comment:
/* For test only.*/ comments are redundant since this class itself is a test code. Let's remove. Let's also shorten the flag names.

```
bool failGetNext,
bool failGetSchema
```

Reply:
Fixed.